src/alert/stories/Alert.stories.tsx

```diff
@@ -45,10 +43,7 @@ const Base: Story<AlertProps> = args => {
         <AlertActionButton>Reach Out</AlertActionButton>
         <Button
           aria-label="close"
-          className={ocx(
-            "h-5 px-0 bg-transparent min-w-5 ml-2",
-            `hover:${buttonBg[status]}`,
-          )}
+          className={`h-5 px-0 bg-transparent min-w-5 ml-2 hover:${buttonBg[status]}`}
```

Comment:
🤔 hopefully it will get properly purged. because we are concatenating here.

Reply:
Hope not 🤔

Reply:
Yeah, Oh well it would not affect too much since it's on storybook anyways.